tardis/transport/montecarlo/tests/conftest.py

```diff
@@ -138,3 +139,21 @@ def random_call():
         np.random.random()
 
     return njit(random_call)
+
+
+@pytest.fixture(scope="package")
+def simulation_rpacket_tracking_enabled(config_verysimple, atomic_dataset):
```

Comment:
think if the fixture name makes sense

Reply:
I removed the virtual packet logging part since it was not required for any of the tests.